storybook/storybook-react/src/components/Button/Button.stories.tsx

```diff
@@ -0,0 +1,55 @@
+/**
+ * @license EUPL-1.2+
+ * Copyright (c) 2023 Gemeente Amsterdam
+ */
+
+import { Button } from '@amsterdam/design-system-react/src'
+import { Meta, StoryObj } from '@storybook/react'
+
+import '@amsterdam/design-system-css/src/button/button.scss'
+
+const meta: Meta<typeof Button> = {
```

Comment:
Can you use the same structure as we do in the heading story of this [this PR](https://github.com/Amsterdam/design-system/pull/224/files#)?

Reply:
done